From 3727bd26987c9052a298adb2f74079605b2afc05 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Thu, 21 Jul 2022 19:35:44 +0300
Subject: [PATCH] [SW-94864] rom: export address valid check api

Export address valid check api for re-use in other FW components.

Change-Id: Ib6f7ab03331c91baa2f1662b77f1a2d7d974395b
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index c5bafc7..a947d0b 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -8,3 +8,4 @@
 security_rom_get_hash
 arcboot_rom_mem_init
 arcboot_rom_get_dma_dest_type
+arcboot_is_valid_address_range
